docs/SETUP.md

```diff
@@ -41,29 +33,25 @@ Download the source (via [git](git@github.com:zendesk/linksf.git) or [.zip file]
 
 ### Secrets
 
-In the project root, you'll find a `.env.example` file. Make a copy of that file called `.env` in the same directory. It will probably look something like this:
+In the project root, you'll find a `.env.example` file. Make copies of that file called `.env.dev` and `.env.prod` in the same directory. They will probably look something like this:
 
 ```
-PARSE_DEV_APP_ID=xxxxxxx
-PARSE_DEV_JS_KEY=xxxxxxx
-PARSE_DEV_MASTER_KEY=xxxxxxx
-PARSE_PROD_APP_ID=xxxxxxx
-PARSE_PROD_JS_KEY=xxxxxxx
-PARSE_PROD_MASTER_KEY=xxxxxxx
+PARSE_APP_NAME=xxxxxxx
+PARSE_APP_ID=xxxxxxx
+PARSE_JS_KEY=xxxxxxx
+PARSE_MASTER_KEY=xxxxxxx
 ...
 ```
 
 There are placeholders for development and production Parse tokens because we don't want any breakage during development to affect the production site. Create a [new Parse app](https://parse.com/apps/new) and call it whatever you'd like. Replace the token placeholder values (`xxxx`) with those specific to your apps from [Parse](https://parse.com/account/keys). The Mailgun and Google Analytics tokens can be ignored for now, if you'd like.
 
-Keep your `.env` file secret (out of source control, etc).
+Keep your `.env.dev` and `.env.prod` files secret (out of source control, etc).
 
 ### Building the site
 
 From the project root (and with the command line tools outlined above installed):
 
-1. `npm install`
```

Comment:
I think your version of npm is old: https://github.com/rlidwka/sinopia/issues/49#issuecomment-36779628

Pretty common these days I think for a relative path; lets you avoid globally installing things (and lets you use different versions of the `grunt` cli). and yes, it will look in your locally installed `node_modules` dir.